packages/orchestration/src/orchestration-types.ts

```diff
@@ -446,13 +445,52 @@ export interface TranslationConfigParams {
   targetLanguage: string;
 }
 
+/**
+ * Output parameters for translation output configuration.
+ */
+export interface TranslationOutputParameters {
```

Comment:
[q] This is possible without code duplication, don't you think? Just a suggestion:
```
type TranslationConfigParams //(same as original)
export type TranslationInputParameters = TranslationConfigParams
export type TranslationOutputParameters = TranslationConfigParams
```

This still allows for future extension with `&`.

Reply:
I modified the Azure Config types, and made some other fixes.